testsuite/domain/src/test/java/org/jboss/as/test/integration/domain/suites/IgnoredResourcesProfileCloneTestCase.java

```diff
@@ -0,0 +1,317 @@
+import org.jboss.as.test.integration.domain.management.util.DomainTestUtils;
+import org.jboss.dmr.ModelNode;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.FixMethodOrder;
+import org.junit.Test;
+import org.junit.runners.MethodSorters;
+import org.xnio.IoUtils;
+
+
+/**
+ * Tests effect of ignoring profiles when profiles are cloned
+ *
+ * @author Kabir Khan
+ */
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
+public class IgnoredResourcesProfileCloneTestCase {
+
+    private static final String ORIGINAL_PROFILE = "default";
+
+    ///The testsuite host-slave.xml is set up to ignore this profile
+    private static final String IGNORED_PROFILE = "ignored";
+
+    private static final String NEW_PROFILE = "clone-profile-test-case-default";
+
+    private static final PathAddress SLAVE_ADDR = PathAddress.pathAddress(HOST, "slave");
+    /**
+     * Domain configuration
+     */
+    private static File domainCfg;
+
+    private static DomainTestSupport testSupport;
+    private static DomainLifecycleUtil masterLifecycleUtil;
+    private static DomainLifecycleUtil slaveLifecycleUtil;
+
+    @BeforeClass
+    public static void beforeClass() throws Exception {
+        testSupport = DomainTestSuite.createSupport(IgnoredResourcesProfileCloneTestCase.class.getSimpleName());
+        masterLifecycleUtil = testSupport.getDomainMasterLifecycleUtil();
+        slaveLifecycleUtil = testSupport.getDomainSlaveLifecycleUtil();
+
+        File masterDir = new File(testSupport.getDomainMasterConfiguration().getDomainDirectory());
+        domainCfg = new File(masterDir, "configuration"
+                + File.separator + "testing-domain-standard.xml");
+    }
+
+    @AfterClass
+    public static void afterClass() throws Exception {
+        testSupport = null;
+        masterLifecycleUtil = null;
+        slaveLifecycleUtil = null;
+        DomainTestSuite.stopSupport();
+    }
+
+    @Test
+    public void test01_ProfileCloneIgnoredWithIgnoreUnusedConfiguration() throws Exception {
+        final DomainClient masterClient = masterLifecycleUtil.getDomainClient();
+        final DomainClient slaveClient = slaveLifecycleUtil.getDomainClient();
+
+        try {
+            final ModelNode originalSlaveDc = DomainTestUtils.executeForResult(
+                    Util.getReadAttributeOperation(SLAVE_ADDR, DOMAIN_CONTROLLER),
+                    masterClient).get(REMOTE);
+            originalSlaveDc.protect();
+            Assert.assertFalse(originalSlaveDc.hasDefined(IGNORE_UNUSED_CONFIG));
+            try {
+                //TODO Remove this - Once the default for ignore-unused-configuration is true, we don't need to set it to true
+                ModelNode newSlaveDc = originalSlaveDc.clone();
+                newSlaveDc.get(IGNORE_UNUSED_CONFIG).set(true);
+                writeSlaveDomainController(slaveClient, newSlaveDc);
+                reloadSlave(slaveLifecycleUtil);
```

Comment:
I plan to ensure Ken's work is merged before dealing with this, as it's more critical, so you should assume you'll remove this before this PR gets merged.